Servers.xml

```diff
@@ -433,3 +433,17 @@
     <discord_url>https://discord.gg/n4T2AhEg</discord_url>
   </ServerItem>
 </ArrayOfServerItem>
+<ArrayOfServerItem>
+  <ServerItem>
+    <id>3f1f41ec-c7fd-4ed9-b47d-25b0d94219c1</id>
+    <name>Unfamiliar Shores</name>
+    <description>Pvp Focused Custom Content CustomDM Client with Modes Softcore, Hardcore Pk/Npk, Solo Self Found</description>
+    <emu>ACE</emu>
+    <server_host>74.50.118.178</server_host>
+    <server_port>9000</server_port>
+    <type>Pk/Npk/Modes</type>
+    <status>Stable potnetial future wipe TBD by server</status>
```

Comment:
```suggestion
    <status>Stable</status>
```

This field should also probably stick to what's at https://github.com/acresources/serverslist?tab=readme-ov-file#server-template-fields. Feel free to update your description to include the information my suggestion removes.